### PR TITLE
Suppress activate/deactivate messages by default

### PR DESCRIPTION
**Pull request by puhitaku**
This PR includes changes that suppress message on virtualenv activation and de-activation.
Detailed info is here: #168 

## What I did

Added `if` statement around echoes in `bin/pyenv-sh-activate` and `bin/pyenv-sh-deactivate`. If users want pyenv to show verbose activate/deactivate message, they have to configure `export PYENV_VIRTUALENV_DISABLE_PROMPT=1`.

### CODE REVIEW

**Comment by puhitaku** [comment]
I've forgot adding appropriate tests. Please wait a little ...

File: bin/pyenv-sh-activate [hunk 1]
```diff
@@ -138,7 +138,9 @@ fi
 
 pyenv-sh-deactivate --force --quiet ${VERBOSE+--verbose} || true
 
-echo "pyenv-virtualenv: activate ${venv}" 1>&2
+if [ -n "$PYENV_VIRTUALENV_VERBOSE_ACTIVATE" ]; then
```
Comment (yyuu) [inline bin/pyenv-sh-activate]: How do you think how we should manage existing `${VERBOSE}` here? In my guts feeling, this line should care about both `${VERBOSE}` and `${PYENV_VIRTUALENV_VERBOSE_ACTIVATE}` too....

**Comment by yyuu** [comment]
CI's still failing. I'll try to look into this once I get a chance....

File: bin/pyenv-sh-activate (hunk 1 above)
Comment (puhitaku) [inline bin/pyenv-sh-activate]: You mean like `if [ -n "$PYENV_VIRTUALENV_VERBOSE_ACTIVATE" -a -n "$VERBOSE" ]` ?

**Comment by puhitaku** [comment]
Sorry for being late but bats' test drives me crazy. I've edited test codes but it doesn't work as I expected (bats says that expected output and actual output are different, but they look same for me.) I'll push (still not working) additional commits later.

File: bin/pyenv-sh-activate (hunk 1 above)
Comment (blueyed) [inline bin/pyenv-sh-activate]: VERBOSE seems to be used with the called tools only?!

**Comment by blueyed** [comment]
@puhitaku 
For the tests, you might want to push what you have (and squash it later), so it can be reviewed.
See #170 btw ("deactivate" message with `pyenv activate` and no active virtualenv (because of `--force`)), which I've came across a while ago myself.

**Comment by puhitaku** [comment]
Yes! It's passing now.

**Comment by blueyed** [comment]
LGTM (from the behavior).
I would like for PYENV_VIRTUALENV_DISABLE_PROMPT=0 to behave logically (i.e. as if unset) though.
So it should check for `!= 0` or `= 1` instead.  But feel free to ignore this.

File: test/conda-activate.bats [hunk 2]
```diff
@@ -32,7 +32,6 @@ setup() {
   assert_success
   assert_output <<EOS
 deactivated
-pyenv-virtualenv: activate anaconda-2.3.0
```
Comment (yyuu) [inline test/conda-activate.bats]: Please add `unset PYENV_VIRTUALENV_VERBOSE_ACTIVATE` in the `setup` of this file.

File: test/conda-deactivate.bats [hunk 3]
```diff
@@ -29,7 +29,6 @@ setup() {
 
   assert_success
   assert_output <<EOS
-pyenv-virtualenv: deactivate anaconda-2.3.0
```
Comment (yyuu) [inline test/conda-deactivate.bats]: Please add `unset PYENV_VIRTUALENV_VERBOSE_ACTIVATE` in the `setup` of this file.

File: test/deactivate.bats [hunk 4]
```diff
@@ -27,7 +27,6 @@ setup() {
 
   assert_success
   assert_output <<EOS
-pyenv-virtualenv: deactivate venv
```
Comment (yyuu) [inline test/deactivate.bats]: Please add `unset PYENV_VIRTUALENV_VERBOSE_ACTIVATE` in the `setup` of this file.

File: test/deactivate.bats [hunk 5]
```diff
@@ -53,6 +52,7 @@ EOS
   export PYENV_VIRTUAL_ENV="${PYENV_ROOT}/versions/venv"
   export VIRTUAL_ENV="${PYENV_ROOT}/versions/venv"
   export PYENV_ACTIVATE_SHELL=
+  export PYENV_VIRTUALENV_VERBOSE_ACTIVATE=1
```
Comment (yyuu) [inline test/deactivate.bats]: Adding test case for `PYENV_VIRTUALENV_VERBOSE_ACTIVATE=1` is nice :+1:

File: bin/pyenv-sh-activate (hunk 1 above)
Comment (yyuu) [inline bin/pyenv-sh-activate]: Basically the `${VERBOSE}` is used mainly from internally. I think just checking `${PYENV_VIRTUALENV_VERBOSE_ACTIVATE}` here could be sufficient at least for now.

**Comment by puhitaku** [comment]
Oh I've forgot to tell that I've pushed commits.
Anyway thanks for merging!

**Pull request merged**